extensions/resource-deployment/src/services/resourceTypeService.ts

```diff
@@ -108,6 +114,34 @@ export class ResourceTypeService implements IResourceTypeService {
 		}
 	}
 
+	private getResourceSubTypes(filterByPlatform: boolean = true, resourceType: ResourceType): void {
+		const resourceSubTypes: ResourceSubType[] = [];
+		vscode.extensions.all.forEach((extension) => {
+			const extensionResourceSubTypes = extension.packageJSON.contributes && extension.packageJSON.contributes.resourceDeploymentSubTypes as ResourceSubType[];
```

Comment:
`const extensionResourceSubTypes = extension.packageJSON.contributes?.resourceDeploymentSubTypes as ResourceSubType[];`

Reply:
(and then you can make it even shorter by removing the if check on the next line and just doing `.resourceDeploymentSubTypes?.forEach(resourceSubType => { ... `